cms/server/admin/templates/participation_preview.html

```diff
@@ -0,0 +1,53 @@
+{% extends base.html %}
+
+{% block core %}
+
+<h1>Import users</h1>
+
+<div id="something">
+  <form enctype="multipart/form-data" action="{{ url_root }}/contest/{{ contest.id }}/users/import" method="POST">
+    <input type="hidden" name="action" value="save">
+    <table class="bordered">
+      <thead>
+        <tr>
+          <th>Participant</th>
+          <th>Username</th>
+          <th>Team</th>
+          {% if load_passwords %}
+          <th>Password</th>
+          {% end %}
+        </tr>
+      </thead>
+      <tbody>
+        {% for u in users %}
+          <tr>
+            {% if u['participant'] %}
+              <td>V</td>
```

Comment:
I think that `Yes`/`No` or `Y`/`N` is better than `V`/`X`. At any rate, Unicode symbols can be used too.